### PR TITLE
Initial population of ASAP website for launch

### PR DESCRIPTION
**Pull request by jchodera**
This is an initial pass at sweeping in [all job ads from the google doc](https://docs.google.com/document/d/1K8UmB-HpTHdpfbDM-vgj8Mna08RnFE0ZmKofzubrGUo/edit) with initial minimal formatting.

I have a lot more work to do on the site, but this should be ready to review and merge, @mc-robinson. You are welcome to test it locally with `hugo server -D`

### CODE REVIEW

File: data/jobs/2022-05-15-vondelft-xchem-compchem-postdoc.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+role: To coordinate structure biology support and structural enablement across projects and cores
```
Comment (mc-robinson) [inline data/jobs/2022-05-15-vondelft-xchem-compchem-postdoc.yaml:1]: There seems to be a mismatch with some ads treating `role` as a noun `Researcher in....` vs these which are more the infinitive `To ...`

I'd suggest we pick one style. After building the site locally. These long phrases `To coordinate structure biology support and structural enablement across projects and cores` do seem a bit of a strange way to start the ad. 
Screen Shot 2022-05-15 at 9 35 35 PM

I'd suggest a change to something like "Researcher coordinating structure biology support and structural enablement across projects and cores"

File: data/jobs/2022-05-15-medchemica-med-chemist.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+duration: 1 year initial contract with opportunity for further permanent position
+availability: September  - December 2022
+status: open
+summary: |
+  [MedChemica](http://www.medchemica.com) is leading the optimisation of new antiviral agents within the ASAP antiviral drug discovery (AViDD) Center. Within our team, we analyse biological and ADMET testing data, and combine that with protein-ligand structural data, FEP calculations and machine learning methods to both design and prioritise compounds for synthesis.  As a member of the lead optimisation team, you will work closely with teams of synthetic chemists, other medicinal chemists, computational chemists, virologists, pharmacokineticists and physical chemists to make sure the best compounds get made and tested.  The lead optimisation team is central in delivering the overall Center goals of generating new anti-virals for human trials ready to treat future pandemics.  You will be working within MedChemica Consultancy Limited where you will be a supportive and supported team member, and where you will also have opportunities to contribute to the development of new AI tools for medicinal chemistry.
```
Comment (mc-robinson) [inline data/jobs/2022-05-15-medchemica-med-chemist.yaml:14]: Can we set a consortium wide standard of American spelling :)

**Review by mc-robinson — COMMENTED** [review]
@jchodera this looks mostly good to me, just brief comments. 

One question. Do we want to push these to prod now? If site gets indexed, we may be a search result before we want to be? Perhaps best to keep in your branch for now?

File: data/jobs/2022-05-15-medchemica-med-chemist.yaml (hunk 2 above)
Comment (jchodera) [reply data/jobs/2022-05-15-medchemica-med-chemist.yaml:14]: Good idea!

**Comment by jchodera** [comment]
> Perhaps best to keep in your branch for now?

OK, I'll consolidate all changes into a single PR we can merge when needed.

File: data/jobs/2022-05-15-vondelft-xchem-compchem-postdoc.yaml (hunk 1 above)
Comment (jchodera) [reply data/jobs/2022-05-15-vondelft-xchem-compchem-postdoc.yaml:1]: We should clean this up to make it consistent, but we might have to do it in a second pass.

We just need bare minimum functionality for now.

**Comment by jchodera** [comment]
This is ready to merge and push once we see the announcement!

**Pull request merged**